eng/tox/allowed_pylint_failures.py

```diff
@@ -56,4 +56,5 @@
     "azure-purview-catalog",
     "azure-messaging-nspkg",
     "azure-agrifood-farming",
+    "azure-eventhub"
```

Comment:
Avoid  block by 'azure-eventhub' for release pipeline

Reply:
We normally enforce `pylint` on track2 packages. I see that we did this to get it released, no worries there. Still filed an issue to re-enable [here](https://github.com/Azure/azure-sdk-for-python/issues/18986) because this is the type of thing that gets forgotten later.